content/en/docs/concepts/workloads/pods/pod-topology-spread-constraints.md

```diff
@@ -302,7 +302,8 @@ apiVersion: kubescheduler.config.k8s.io/v1beta3
 kind: KubeSchedulerConfiguration
 
 profiles:
-  - pluginConfig:
+  - schedulerName: default-scheduler
```

Comment:
schedulerName does appear to be required. See https://kubernetes.io/docs/reference/config-api/kube-scheduler-config.v1beta3/#kubescheduler-config-k8s-io-v1beta3-KubeSchedulerProfile

Great job!